_episodes/03-skimage-images.md

```diff
@@ -396,22 +398,23 @@ determines the backend to use based on the image type.
 > > ~~~
 > > import imageio.v3
 > >
-> > image = iio.imread(uri="data/sudoku.jpg", mode="L")
+> > sodoku = iio.imread(uri="data/sudoku.png")
```

Comment:
This was updated in PR #253 (hence the conflict). Would you mind merging `gh-pages` to sync up your branch? Thanks!